### DISCUSSION TITLE
My PP went down on the main site as soon as i closed osu! lazer

### DISCUSSION THREAD

**Discussion by nish4eva** [ppy/osu#23313] [Q&A] [answered]
<img width="347" alt="image" src="https://user-images.githubusercontent.com/70297729/234371832-869d2200-5202-4e4d-bc4b-c6289b110abd.png">
this number was 200 on my profile info before i closed osu lazer

**Comment by peppy** [comment, marked as answer]
Lazer doesn't touch your PP value on the non-lazer site for now. You must be mistaken in one way or another.

**Reply by nish4eva** [reply]
i was talking about the pp shown within the lazer game, the one in player info, it doesnt show the same value once i restart the game. However, recently played is visible
